annals-of-botany.csl

```diff
@@ -104,6 +104,13 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <group suffix=". ">
+            <text macro="title" font-weight="italics" prefix=" "/>
+          </group>
+          <text variable="url" suffix=". "/>
+          <date variabiable="accessed"/>
+        </else-if>    
```

Comment:
@damnation333, see the validation results: http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fdamnation333%2Fstyles%2Fpatch-8%2Fannals-of-botany.csl&version=1.0.1

To use italics, use `font-style="italic"`. To print the URL, use the "URL" (all-caps) variable. And change "variabiable" to "variable".